tally_ho/libs/verify/quarantine_checks.py

```diff
@@ -97,6 +101,59 @@ def pass_tampering(result_form):
     return diff <= scaled_tolerance
 
 
+def pass_ballots_number_validation(result_form):
+    """Validate that the total number of received ballots equals the
+    total of the ballots inside plus ballots outside.
```

Comment:
what are ballots `outside`?

Reply:
Same here, `ballots inside the box` and `ballots outside the box`.

Reply:
Done